### PR TITLE
Try to improve table readability

### PR DESCRIPTION
**Pull request by MTRNord**
I am not sure if this is better or worse tbh. The ellipsis via CSS dynamically only works for a single line which is also not great, hence done in R which equally is not great. 

Additionally, the scrollbar imho happens way too late in the table as well. But since it's a table cell, it's not possible to override it's cell height. 

There are ways to use grid + divs but this is not a div table and tbh accessiblity-wise that's not great. 

TLDR: Table CSS still sucks in 2023 but at least this doesn't massively overflow anymore and it scrolls on mobile.

image

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
This looks good, thanks @MTRNord! I'll try to give it a test tonight just for sanity, and then I can merge it.
